no-OS/zynqmp_ddr_patch.sh

```diff
@@ -1,125 +1,129 @@
 #!/bin/bash
+#
```

Comment:
maybe squash this commit into the previous one;
no need to keep fix-commits in the master branch;